flare_flutter/lib/flare_actor.dart

```diff
@@ -72,13 +86,46 @@ class FlareActor extends LeafRenderObjectWidget {
     this.shouldClip = true,
     this.sizeFromArtboard = false,
     this.artboard,
-  });
+  }) : flareProvider = null;
+
+  FlareActor.rootBundle(
```

Comment:
@ybbaigo I changed the syntax slightly here so you can do ```FlareActor.rootBundle('assets/my.flr')``` and ```FlareActor.asset(AssetFlare(...))```. You can see how I use it in [example/simple/lib/main.dart](https://github.com/2d-inc/Flare-Flutter/blob/6a1d0649c455ee76cc0e8789ca2b9364aa7cfba5/example/simple/lib/main.dart#L32)